libraries/AP_Camera/json/video_stream_information.json

```diff
@@ -0,0 +1,20 @@
+# This file is used to populate the VIDEO_STREAM_INFORMATION message sent by
```

Comment:
SHould we put these under an "examples" subdirectory as we do for a lot of other things?

Reply:
I'd prefer they live in the `AP_Camera` folder. The code in the backend makes assumptions about what fields are and aren't present in the JSON.